evergreen.yml

```diff
@@ -263,6 +269,14 @@ tasks:
       - func: "fetch signed connector"
       - func: "upload release"
 
+  - name: test-release
```

Comment:
This task doesn't seem to be called anywhere.  Perhaps it was from your testing?

Reply:
This is meant to be manually triggered. I found it invaluable when adding a variant for mongosql-rs. If we'll never have another variant (like a 32 bit build), then we don't necessarily need this.

Reply:
Ah okay, thanks for the explanation.  Let's leave this in  👍